acceptance/terrafarm/farmer.go

```diff
@@ -131,7 +132,7 @@ func (f *Farmer) CollectLogs() {
 		dest += strconv.Itoa(i)
 		if err := f.scp(host, f.defaultKeyFile(), src,
 			filepath.Join(f.AbsLogDir(), dest)); err != nil {
-			fmt.Fprintf(os.Stderr, "error collecting %s from host %s: %s", src, host, err)
+			f.logf("error collecting %s from host %s: %s", src, host, err)
```

Comment:
I haven't looked at this code before. Is there a reason it isn't using `log.V(1).Infof` or something like that? Then we could control the output from the command line via `-vmodule`.